app/views/course/user_invitations/_course_user_invitation.html.slim

```diff
@@ -9,7 +11,7 @@
     td = format_datetime(invitation.sent_at, :short) if invitation.sent_at
     td
       div.btn-group
-        = link_to course_user_invitation_resend_invitation_path(current_course, invitation),
+        = link_to course_user_invitation_resend_invitation_path(current_course, invitation, serial_number: serial_number),
```

Comment:
We need to pass serial_number here since it will only update the affected row using ajax for resend_invitation. The serial_number will then be passed back to the partial to be rendered.